.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: scala
 jdk:
 - oraclejdk8
-script: sbt +test
+script: 
+  - sbt +test
```

Comment:
I don't want to redundantly specify the Scala version for Travis. All we need is `crossScalaVersions` in the build and the use the `+`.